src/options.c

```diff
@@ -98,6 +98,13 @@ static void usage(const char *argv0, int ret) {
 	    "--animation-clamping\n"
 	    "  Whether to clamp animations (default: true)\n"
 	    "\n"
+	    "--animation-opacity-min (default: 0.0).\n"
+	    "  Minim Opacity of animation processing.\n"
+	    "  If you hope disable animation opacity, you can set this parameter to 1.0.\n"
```

Comment:
Again, bad english. How about "If you wish to disable window blending opacity during animation completely, you can set this parameter to 1.0"?

Also, I'm not sure it's a great idea to suggest doing that to disable animation opacity, since an extra pixmap will still drawn undernearth. It would probably be better to add a separate config option for that.

Reply:
You are right. maybe need a separate config option for that.